packages/frontend/src/lib/progress/TaskElement.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
```

Comment:
This could have a been a separate PR

Reply:
I have created a PR for this: https://github.com/projectatomic/ai-studio/pull/296